packages/terra-paginator/src/terra-dev-site/doc/example/ProgressivePaginatorWithoutTotalCountExample.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Dialog from 'terra-dialog';
+import ProgressivePaginator from 'terra-paginator/lib/ProgressivePaginator';
+import classNames from 'classnames/bind';
+import styles from './PaginatorExampleCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const fillArray = (value, len) => {
+  const arr = [];
+  for (let i = 0; i < len; i += 1) {
+    arr.push(<p key={Math.floor(Math.random() * Math.floor(100000))}>{value}</p>);
+  }
+  return arr;
+};
+
+const buildPage = () => {
+  let fullContent = [];
+  const content = ('Lorem ipsum dolor sit amet, consectetur adipiscing elit. '
+    + 'Fusce porttitor ullamcorper nisi, vel tincidunt dui pharetra vel. '
+    + 'Morbi eu rutrum nibh, sit amet placerat libero. Integer vel dapibus nibh. '
+    + 'Donec tempor mi vitae lorem congue, ut ultrices metus feugiat. Sed non commodo felis. '
+    + 'Aliquam eget maximus dui, ut rhoncus augue.');
+
+  fullContent = fillArray(content, 10);
+
+  return (
+    fullContent
+  );
+};
+
+class InfiniteProgressivePaginatorExample extends React.Component {
```

Comment:
Can we convert this new example to use hooks?

Reply:
updated [here](https://github.com/cerner/terra-core/pull/3190/commits/d0e5ef41693dba34291910c0e38aeb195fc36a43#diff-c90ec0cfc9f5faa49aff0b98f70e2b2d0d110679caf110c818c44f9d5c31a045)